app/packages/assistant_api/doc/ClientInfoSchema.md

```
@@ -0,0 +1,20 @@
+# assistant_api.model.ClientInfoSchema
+
+## Load the model package
+```dart
+import 'package:assistant_api/api.dart';
+```
+
+## Properties
+Name | Type | Description | Notes
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)